evap/results/exporters.py

```diff
@@ -232,15 +255,20 @@ def write_overall_results(self, evaluations_with_results, course_results_exist):
                     self.write_cell()
                     continue
 
-                avg = evaluation.course.avg_grade
+                avg = evaluation.course.avg_grade  # type: ignore[attr-defined]
                 style = self.grade_to_style(avg) if avg is not None else "border_left_right"
                 self.write_cell(avg, style)
             self.next_row()
 
             # Same reasoning as above.
             self.write_empty_row_with_styles(["default"] + ["border_top" if gt1 else "default" for gt1 in count_gt_1])
 
-    def write_questionnaire(self, questionnaire, evaluations_with_results, contributor):
+    def write_questionnaire(
+        self,
+        questionnaire: Questionnaire,
+        evaluations_with_results: list[tuple[Evaluation, Any]],
```

Comment:
The code below looks to me as if `Any` here should be `dict[int, GradeResult]`. Is that correct? If yes, I'd rather that annotated.